src/synopsys-action/tools-parameter.ts

```diff
@@ -23,20 +23,37 @@ export interface PolarisData {
   assessment: {types: PolarisAssessmentType[]}
 }
 
+export interface Coverity {
+  coverity: CoverityConnect
+}
+
+export interface CoverityConnect {
+  connect: CoverityData
+}
+
+export interface CoverityData {
+  user: {name: string; password: string}
+  url: string
+  project: {name: string}
+  policy: {view: string}
+}
+
 export class SynopsysToolsParameter {
   tempDir: string
   private static STAGE_OPTION = '--stage'
   private static STATE_OPTION = '--state'
   private static POLARIS_STAGE = 'polaris'
   private static STATE_FILE_NAME = 'input.json'
+  // Coverity parameters
+  private static COVERITY_STAGE = 'connect'
   private static SPACE = ' '
 
   constructor(tempDir: string) {
     this.tempDir = tempDir
   }
 
-  getFormattedCommandForPolaris(accessTok: string, applicationName: string, projectName: string, serverURL: string, assessmentTypes: string[]): string {
-    if (accessTok == null || accessTok.length === 0 || applicationName == null || applicationName.length === 0 || projectName == null || projectName.length === 0 || serverURL == null || serverURL.length === 0 || assessmentTypes.length === 0) {
```

Comment:
cant we have a utility method for null n empty check?

Reply:
done